src/javaRosa.js

```diff
@@ -1980,6 +1973,24 @@ define([
                 widget: iTextIDWidget,
                 widgetValuePath: "constraintMsgItext"
             };
+            databind.constraintMediaIText = function (mugOptions) {
+                return mugOptions.isSpecialGroup ? undefined : {
```

Comment:
it's a little dangerous to reference `undefined` by name like this because it can be globally reassigned (is not guaranteed to actually have an undefined value). A very unhandy feature of JavaScript. Probably better to use `null`.